rtgui/editorpanel.cc

```diff
@@ -46,6 +46,8 @@ class EditorPanel::MonitorProfileSelector
     void prepareProfileBox ()
     {
         profileBox.set_size_request (100, -1);
+        profileBox.setPreferredWidth(90, 150);
```

Comment:
I was under the impression we need have a space before parentheses for function and constructor calls, e.g. `profileBox.setPreferredWidth (90, 150);`? I mess it up myself all the time, but the code added here seems inconsistent w.r.t. to this?